### PR TITLE
Update kronos docker

### PR DESCRIPTION
**Pull request by lydiagu**
Updated build to use docker_tools/manage_docker.py.
Cleaned up the Dockerfile.

### CODE REVIEW

File: kronos/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:14.04
+
+RUN \
+  apt-get update && \
+  apt-get install -y python2.7 build-essential wget supervisor \
+  libevent-dev libpcre3-dev python-all-dev python-pip uuid-dev
+COPY chronology/kronos/requirements.txt requirements.txt
+RUN sudo pip install -r requirements.txt
```
Comment (marcua) [inline kronos/docker/Dockerfile]: note that the other dockerfiles put stuff in /requirements.txt and /chronology explicitly.  for consistency with them, it makes sense to explicitly specify the root dir.

i'm also a docker n00b, so perhaps you know something i don't about how that explicitness doesn't matter :)

File: kronos/docker/README.md [hunk 2]
```diff
@@ -8,9 +8,9 @@ We've already pushed a working [Dockerfile to Docker Hub](https://registry.hub.d
...
   * Run `sudo docker run -d -p 8150:8150 -v /etc/kronos:/etc/kronos -v /var/log/kronos:/var/log/kronos chronology/kronos:$(KRONOS_VERSION)`
 
-## How to create new Dockerfiles for future versions of Kronos
+## How to build/push future versions
 
-  * Tag the git commit for which you want to generate a Docker.
-  * In `kronos/docker` directory, run `KRONOS_VERSION=v0.7.0 make generate_dockerfile`
-  * In the same directory, run `KRONOS_VERSION=v0.7.0 make build`. Note: Docker aggressively caches commands for performance.  To avoid this, for example if you had to change which hash a tag points to, call `KRONOS_VERSION=v0.7.0 make build_nocache`.
-  * In the same directory, run `KRONOS_VERSION=v0.7.0 make push`
+  * `./build.sh [git hash or tag you want to build] chronology/jia:v[version]`
```
Comment (marcua) [inline kronos/docker/README.md]: jia->Kronos

Comment (marcua) [inline kronos/docker/README.md]: jia->kronos

**Comment by marcua** [comment]
nitpick review, but almost there!

File: kronos/docker/Dockerfile (hunk 1 above)
Comment (lydiagu) [inline kronos/docker/Dockerfile]: done

File: kronos/docker/README.md (hunk 2 above)
Comment (lydiagu) [inline kronos/docker/README.md]: oops! fixed

File: kronos/docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:14.04
+
+RUN \
+  apt-get update && \
+  apt-get install -y python2.7 build-essential wget supervisor \
```
Comment (silasbw) [inline kronos/docker/Dockerfile]: i'd put these packages each on their own line and sort them alphanumerically:

  https://docs.docker.com/articles/dockerfile_best-practices/

File: docker_tools/manage_docker.py [hunk 4]
```diff
@@ -7,11 +7,15 @@
 import argparse
 import logging
 import os
+import sys
 
 log = logging.getLogger(__name__)
 
 def main(args):
   if args.subcommand == 'export':
+    found_hash = os.system('git show {hash}'.format(hash=args.hash))
```
Comment (silasbw) [inline docker_tools/manage_docker.py]: subprocess.check_call, yo.

**Comment by marcua** [comment]
LGTM.  @silasbw has good comments, and I'll leave it to him to sign off on those.

**Pull request merged**